dpgen2/entrypoint/submit.py

```diff
@@ -184,10 +184,11 @@ def make_conf_list(
         conf_list = []
         for ii in conf_list_fname:
             ss = dpdata.System(ii, type_map=type_map, fmt=fmt)
-            ss.to('lammps/lmp', 'tmp.lmp')
-            conf_list.append(Path('tmp.lmp').read_text())
-        if Path('tmp.lmp').is_file():
-            os.remove('tmp.lmp')
+            if fmt == 'vasp/poscar':
```

Comment:
Why do we need to check the `fmt`? It seems that other format other than `vasp/poscar' would work.

Reply:
Indeed, checking `fmt` is unnecessary, I had overlooked this.